### ISSUE TITLE
If a commented piece of code contains the word "raise" docstring tries to parse it.

### ISSUE THREAD

**Issue by bastienboutonnet** [NilsJPWerner/autoDocstring#143]
**Describe the bug**
I had a function in which I had a `# TODO:` comment that was `raise appropriate error`

When generating the docstring it was parsed as if I was declaring an Exception 😂. Pretty funny corner case! 

Running 0.5.3

**Original Code (with line to generate on):**
```python
def foo(bar):
	print('Hello World')
	# TODO: raise appropriate error down the line
```

**Expected Result:**
```python
def foo(bar):
    """[summary]

    Args:
        bar ([type]): [description]
    """
	print('Hello World')
	# TODO: raise appropriate error down the line
```

**Actual Result:**
```python
def foo(bar):
    """[summary]

    Args:
        bar ([type]): [description]

    Raises:
        appropriate: [description]
    """
	print('Hello World')
	# TODO: raise appropriate error down the line
```

**Comment by bastienboutonnet** [comment]
@NilsJPWerner I'm tempted to modify the regex with requiring that "raise" be the first thing that starts the string and (maybe trim the line so that leading whitespace from indentation doesn't mess with is)

https://github.com/bastienboutonnet/autoDocstring/blob/master/src/parse/parse_parameters.ts#L139

would look like:
```typescript
function parseExceptions(body: string[]): Exception[] {
    const exceptions: Exception[] = [];
    const pattern = /^raise\s+([\w.]+)/;

    for (const line of body) {
        const match = line.trim.match(pattern);

        if (match == null) {
            continue;
        }

        exceptions.push({ type: match[1] });
    }

    return exceptions;
}
```

**Comment by bastienboutonnet** [comment]
Ah! I found a way to regex it better actually 🎉 so my concern with the proposed solution to requiring `raise` to start the string is not moot.

Implemented in #144 

Feedback welcome!
